### PR TITLE
Feature/four 9924 data tree visualization on the data connector

### PR DESCRIPTION
**Pull request by gustavobascope**
## Issue & Reproduction Steps
Data Connectors - Add to Response Body tab

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-9924

ci:deploy
ci:package-data-sources:feature/FOUR-9924
.

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8c859fadfe.eng.processmaker.net

File: resources/js/bootstrap.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ import Vue from 'vue'
 import VueCookies from 'vue-cookies'
 import PmqlInput from "./components/shared/PmqlInput.vue";
 import GlobalSearch from "./components/shared/GlobalSearch.vue";
+import DataTreeToggle from "./components/common/data-tree-toggle.vue";
+import TreeView from "./components/TreeView.vue";
```
Comment (nolanpro) [inline resources/js/bootstrap.js:31]: @gustavobascope If we are going to include this globally can we remove it from resources/js/requests/show.js and resources/js/tasks/show.js ?

**Review by nolanpro — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8c859fadfe.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8c859fadfe.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8c859fadfe.eng.processmaker.net

**Review by nolanpro — APPROVED** [review]

**Pull request merged**